REQUIRE

```diff
@@ -2,3 +2,4 @@ julia 0.4
 TaylorSeries 0.3.0
 FactCheck 0.4.2
 Compat 0.9.1
+ValidatedNumerics 0.5.0
```

Comment:
Pushed a new commit where I moved the ValidatedNumerics requirement to test/REQUIRE; tests work fine on my machine with this new change, let's see what the opinion of travis is about this